Assets/Scripts/ROS Scripts/ROS Connections v2/Sensors/PCFaceSensor_ROSSensorConnection.cs

```diff
@@ -68,6 +70,10 @@ void Update()
         }
     }
 
+    public string GetSensorName()
+    {
+        return sensorName;
```

Comment:
Do you want to add this directly into the sensor interface? seems like you adding it for everyone.